config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
   notary:
-    deposit_amount: 100000001
-    deposit_duration: 1001
+    deposit_amount: 100000001  # notary deposit GAS amount
+    deposit_duration: 1001  # notary deposit expiration in blocks
 
 mainchain:
-  dial_timeout: 30s
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for main chain NEO RPC client connection
+  rpc_endpoint:  # main chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.n3.nspcc.ru:30341
     - https://rpc2.n3.nspcc.ru:30341
 
 apiclient:
-  dial_timeout: 15s
+  dial_timeout: 15s  # timeout for NEOFS API client connection
 
 policer:
-  head_timeout: 15s
+  head_timeout: 15s  # timeout for the Policer HEAD remote operation
 
 replicator:
-  put_timeout: 15s
+  put_timeout: 15s  # timeout for the Replicator PUT remote operation
 
 object:
   put:
-    pool_size_remote: 100
-    pool_size_local: 101
+    pool_size_remote: 100  # numbers of async workers for remote PUT operations
+    pool_size_local: 101  # numbers of async workers for local PUT operations
 
 storage:
-  shard_num: 2
+  shard_num: 2  # total shards amount
   shard:
     0:
-      refill_metabase: false
+      refill_metabase: false  # sync metabase with blobstore on start, expensive, leave false until complete understanding
 
-      use_write_cache: false
+      use_write_cache: false  # use write-cache
 
       writecache:
-        path: tmp/0/cache
-        mem_size: 2147483648
-        small_size: 16384
-        max_size: 134217728
-        workers_number: 30
-        size_limit: 3221225472
+        path: tmp/0/cache  # write-cache root directory
+        mem_size: 2147483648  # coarse RAM usage limit for "small" objects
+        small_size: 16384 # size threshold for objects that are considered to be "small"
+        max_size: 134217728  # objects that are bigger than <max_size> bypass write-cache and go to the storage directly
+        workers_number: 30  # thread number of write-cache flushers
+        size_limit: 3221225472  # coarse write-cache total size
 
       metabase:
-        path: tmp/0/meta
-        perm: 0644
+        path: tmp/0/meta  # metabase path
+        perm: 0644  # permissions for metabase files(directories: +x for current user and group)
 
       blobstor:
-        path: tmp/0/blob
-        perm: 0644
-        compress: true
-        shallow_depth: 5
-        small_size_limit: 102400
+        path: tmp/0/blob  # blobstor path
+        perm: 0644  # permissions for blobstor files(directories: +x for current user and group)
+        compress: true  # zstd, level 3, unconfigurable right now
+        shallow_depth: 5 # directory structure depth
+        small_size_limit: 102400  # size threshold for objects that are considered to be "small"
 
         blobovnicza:
-          size: 4194304
-          shallow_depth: 1
-          shallow_width: 4
-          opened_cache_size: 50
+          size: 4194304  # coarse one db size limit, total size: size*width^(depth+1)
```

Comment:
total size will be: size*width^(depth+1)

Reply:
changed